tools/android/packaging/Makefile.in

```diff
@@ -102,7 +102,7 @@ python: | xbmc/assets
 	cd xbmc/assets/python2.7/lib/python2.7/; rm -rf test config lib-dynload
 
 res:
-	mkdir -p xbmc/res xbmc/res/raw xbmc/res/values images
+	mkdir -p xbmc/gen/@APP_PACKAGE_DIR@ xbmc/res xbmc/res/raw xbmc/res/values images
```

Comment:
semi-no. You need the gen but not the APP_PACKAGE_DIR
You can hardcode APP_PACKAGE_DIR to org/xbmc/kodi